pipeline-runner/tests/testthat/test-gem2s-2-load_cellranger.R

```diff
@@ -8,7 +35,7 @@ test_that("format_annot keeps unique rows", {
         sample2 = data.frame(ENSID = 1:5, SYMBOL = paste0('gene', 1:5))
     )
 
-    annot <- format_annot(annot_list)
+    annot <- pipeline:::format_annot(annot_list)
```

Comment:
I can't run the tests locally, don't know what I'm doing wrong, how should I install the pipeline package?

Reply:
open RStudio project and type `ctrl+shift+B` or go to the **Build** tab and click _Install and Restart_